### PR TITLE
Updated generate-data-project md with additional Git help

### PR DESCRIPTION
**Pull request by Adam-Tahir-Ensono**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

Added additional git info and commands if github repo created before running scaffold command

#### 🤔 Why

User may run into Git issues if creating repo before running scaffold commands, these steps will ensure success

#### 🛠 How

Documentation update

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [x ] Documentation has been updated to reflect the changes?
- [x ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [x ] Meeting the Coding Standards?

### CODE REVIEW

**Comment by Adam-Tahir-Ensono** [comment]
/ azp run Ensono.stacks-azure-data (2)

File: docs/workloads/azure/data/getting_started/generate_project.md [hunk 1]
```diff
@@ -80,6 +80,29 @@ git push -u origin main
 
 ```
 
+If you have already created a repo in GitHub and want to push this project to the repo then follow this:
```
Comment (adurkan-amido) [inline docs/workloads/azure/data/getting_started/generate_project.md:83]: Could this not be incorporated into the snippet above? I believe the above code is based on the repo already existing also?

**Pull request closed without merging**